unstructured/ingest/cli/cmds/salesforce.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+import click
+
+from unstructured.ingest.cli.common import (
+    add_recursive_option,
+    add_shared_options,
+    log_options,
+    map_to_processor_config,
+    map_to_standard_config,
+    run_init_checks,
+)
+from unstructured.ingest.logger import ingest_log_streaming_init, logger
+from unstructured.ingest.runner import salesforce as salesforce_fn
+
+
+@click.command()
+@click.option(
+    "--salesforce-categories",
+    default=None,
+    required=True,
+    help="Comma separated list of categories to download. "
+    "Do not specify subfolders. Use quotes if spaces in folder names.",
+)
+@click.option(
+    "--salesforce-username",
+    required=True,
+    help="salesforce username bla bla.",
+)
+@click.option(
+    "--salesforce-consumer-key",
+    required=True,
+    help="password bla bla",
+)
+@click.option(
+    "--salesforce-private-key-path",
+    required=True,
+    help="token bla bla.",
```

Comment:
since this is visible in the cli with `unstructured-ingest salesfoce --help` lets fill in some better help text here :)

Reply:
Yep. Bla bla bla is on the vague side.